ee/codegen/src/__test__/nodes/__snapshots__/inline-prompt-node.test.ts.snap

```diff
@@ -89,7 +89,11 @@ class PromptNode(InlinePromptNode):
                         PlainTextPromptBlock(
                             state="ENABLED",
                             cache_config=None,
-                            text="Summarize the following text:\\n\\n",
+                            text="""\\
```

Comment:
The double `\\` is purely an artifact of snapshot testing. When the files get written (as you can see in the updated fixtures below), only a single `\` is used.